rest/taskana-rest-spring-example-common/src/test/java/pro/taskana/RestConfiguration.java

```diff
@@ -0,0 +1,75 @@
+package pro.taskana;
+
+import com.fasterxml.jackson.databind.cfg.HandlerInstantiator;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+import org.springframework.http.converter.json.SpringHandlerInstantiator;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.rest.ldap.LdapClient;
+import pro.taskana.monitor.api.MonitorService;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.workbasket.api.WorkbasketService;
+
+/** Configuration for REST service. */
+@Configuration
+@ComponentScan
+@EnableTransactionManagement
+public class RestConfiguration {
```

Comment:
Do we really need this class here? Can't we use the RestConfiguration from taskana-rest-spring module?